test/interface/mass_matrix_tests.jl

```diff
@@ -76,12 +76,12 @@ dependent_M2 = DiffEqArrayOperator(ones(3,3),update_func=update_func2)
     println("BDFs")
     @test _norm_dsol(ABDF2(),prob,prob2) ≈ 0 atol=1e-7
 
-    @test_skip _norm_dsol(QBDF1(),prob,prob2) ≈ 0 atol=1e-7
+    @test _norm_dsol(QBDF1(),prob,prob2) ≈ 0 atol=1e-7
     @test _norm_dsol(QBDF2(),prob,prob2) ≈ 0 atol=1e-7
-    @test_skip _norm_dsol(QBDF(),prob,prob2) ≈ 0 atol=1e-7
```

Comment:
if qndf1 is the only one effected, why is this changed?

Reply:
QBDF was passing also before